Changes

```diff
@@ -1,6 +1,20 @@
 Release history for Zonemaster component Zonemaster-CLI
 
 
+v7.2.0 2025-03-04 (part of Zonemaster v2024.2.1 release)
+
+ [Release information]
+ - Translations have not been fully updated in this release. They will
+   be updated in an upcoming extra release.
+
+ [Features]
+- Adds translation to Slovenian language (#384)
+
+ [Fixes]
+- Updates translations (#418, #425, #415, #422, #414)
+- Removes duplicate function calls (#420)
```

Comment:
I don't recall if this was a bug fix or just a code cleanup. If it's a bug fix it would be nice to mention something about its nature.